dd-sdk-android/src/main/kotlin/com/datadog/android/v2/core/internal/DatadogContextProvider.kt

```diff
@@ -74,10 +74,14 @@ internal class DatadogContextProvider(val coreFeature: CoreFeature) : ContextPro
                     )
                 },
                 trackingConsent = coreFeature.trackingConsentProvider.getConsent(),
-                featuresContext = emptyMap()
+                featuresContext = coreFeature.featuresContext
             )
         }
 
+    override fun setFeatureContext(feature: String, context: Map<String, Any?>) {
```

Comment:
Looks a bit weird to have this function in the `ContextProvider`. Why not having it in the `SdkCore` ?